AI-and-Analytics/Features-and-Functionality/IntelPython_GPU_numba-dpex_Genetic_Algorithm/README.md

```diff
@@ -0,0 +1,117 @@
+If the default path is used during the installation of Intel® Distribution for Python*:
+```
+source $HOME/intelpython3/bin/activate
+```
+If a non-default path is used:
+```
+source <custom_path>/bin/activate
+```
+
+**3. Clone the GitHub repository**
+<!-- for oneapi-samples: git clone https://github.com/oneapi-src/oneAPI-samples.git
+cd oneAPI-samples/AI-and-Analytics/<samples-folder>/<individual-sample-folder> -->
+<!-- for migrated samples - provide git clone command for individual repo and cd to sample dir --> 
+``` 
+git clone https://github.com/oneapi-src/oneAPI-samples.git
+cd oneAPI-samples/AI-and-Analytics/Features-and-Functionality/IntelPython_GPU_numba-dpex_Genetic_Algorithm
+```
+
+**4. Install dependencies**
+<!-- It is required to have requirement.txt file in sample dir. It should list additional libraries, such as matplotlib, ipykernel etc. -->
+>**Note**: Before running the following commands, make sure your Conda environment is activated
+
+```
+pip install -r requirements.txt
+pip install notebook
+``` 
+For Jupyter Notebook, refer to [Installing Jupyter](https://jupyter.org/install) for detailed installation instructions.
+
+## Run the Sample
+>**Note**: Before running the sample, make sure [Environment Setup](https://github.com/oneapi-src/oneAPI-samples/tree/master/AI-and-Analytics/Features-and-Functionality/IntelPython_GPU_numba-dpex_Genetic_Algorithm#environment-setup) is completed.
+
+### Intel® Distribution for Python* Offline Installer (Validated)
+
+**1. Register Conda kernel to Jupyter Notebook kernel**
+
+If the default path is used during the installation of Intel® Distribution for Python*:
+```
+$HOME/intelpython3/bin/python -m ipykernel install --user --name=base
+```
+If a non-default path is used:
+```
+<custom_path>/bin/python -m ipykernel install --user --name=base
+```
+**2. Launch Jupyter Notebook** 
+<!-- add other flags to jupyter notebook command if needed, such as port 8888 or allow-root -->
+```
+jupyter notebook --ip=0.0.0.0
+```
+**3. Follow the instructions to open the URL with the token in your browser**
+
+**4. Select the Notebook**
+<!-- add sample file name -->
+```
+IntelPython_GPU_numba-dpex_Genetic_Algorithm.ipynb
+```
+**5. Change the kernel to `base`**
+  <!-- specify relevant kernel name(s), for example `pytorch` -->
+**6. Run every cell in the Notebook in sequence**
+
+## Example Output
+
+If successful, the sample displays `[CODE_SAMPLE_COMPLETED_SUCCESSFULLY]` at the end of execution. The sample will print out the runtimes and charts of relative performance with numba-dpex and without any optimizations as the baseline. Additionally, sample will print the best and worst path found in the Traveling Salesman Problem.
+
+## Related Samples
+
+* [Get Started with the Intel® Distribution for Python*](https://www.intel.com/content/www/us/en/developer/articles/technical/get-started-with-intel-distribution-for-python.html)
```

Comment:
Usually in this section we provide the links to other samples from the repo. In this case, it is https://github.com/oneapi-src/oneAPI-samples/tree/master/AI-and-Analytics/Features-and-Functionality/IntelPython_Numpy_Numba_dpex_kNN. This way user will stay on GitHub instead of going to IDZ and then returning back, which makes user path easier